### ISSUE TITLE
Refactor OncoKBConnector into small modules

### ISSUE THREAD

**Issue by zhx828** [cBioPortal/cbioportal#620]
- [ ] Separate all utils into OncoKB.utils singleton
- [ ] Maximize the MVC

Related to #572

**Comment by jjgao** [comment]
@zhx828 what's the status of this issue?

**Comment by zhx828** [comment]
Refactoring will start after #774 being merged.

**Comment by jjgao** [comment]
@zhx828 what's the plan on this?

**Comment by zhx828** [comment]
@jjgao postponed until the data manager is released.

**Comment by jjgao** [comment]
@zhx828 we should consider have a separate repo for the oncokb js client.
